src/fields/config/types.ts

```diff
@@ -335,8 +335,8 @@ export type RelationshipField = FieldBase & {
    * @deprecated Use 'maxRows' instead
    */
   max?: undefined
-  minRows?: number
-  maxRows?: number
+  minRows?: undefined
```

Comment:
Why is the type `undefined` instead of `number`?

Reply:
@DanRibbens If `hasMany` is not set or false, we don't want to allow the user to set the `minRows`/`maxRows` property, as it's useless then.

Sadly, typescript doesn't allow completely hiding it, since it's an optional property. BUT by setting it to undefined, you'll get an error if you try to use `minRows`/`maxRows` without `hasMany` being true.